xcode/Subconscious/Shared/Services/AddressBookService.swift

```diff
@@ -54,6 +56,32 @@ class AddressBookService {
             .eraseToAnyPublisher()
     }
     
+    func followUser(did: Did, petname: Petname) throws {
+        try setPetname(did: did, petname: petname)
+        let version = try self.noosphere.save()
+        try database.writeMetadatadata(key: .sphereVersion, value: version)
```

Comment:
Unsure if writing directly to the database like this is encouraged, or if we should instead move these methods to `NoosphereService` which already manages `.sphereVersion`?

Reply:
I think this is fine for now. We could look later into whether the relationship between services should be refactored in #462.